client/client.cpp

```diff
@@ -186,10 +192,6 @@ bool Client::receive(std::vector<std::string>& updates) {
     error_ = "No active connection";
     return false;
   }
-  if (!conn_->poll(30000)) {
-    error_ = "Timeout while waiting for server";
-    return false;
-  }
```

Comment:
Since the timeout option replaces the previous polling I think we should set the default timeout to 30s to be compatible?